recipes/sml/all/conanfile.py

```diff
@@ -0,0 +1,37 @@
+from conans import ConanFile, tools
+from conans.tools import check_min_cppstd
+import os
+
+
+class SMLConan(ConanFile):
+    name = "sml"
+    homepage = "https://github.com/boost-ext/sml"
+    description = "SML: C++14 State Machine Library"
+    topics = ("state-machine", "boost", "metaprogramming", "design-patterns", "sml")
+    license = "BSL-1.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    settings = "compiler"
+    no_copy_source = True
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
```

Comment:
```suggestion

    def requirements(self):
        self.requires("boost/1.75.0")

```

Reply:
Why do you suggest adding boost? I do not think that the package depends on boost at all. @kris-jusiak, could you please confirm?

Reply:
Because of this line in CMakeLists: https://github.com/boost-ext/sml/blob/b6b1008a31b67479bcd99bf31e36afea1dd4df6f/CMakeLists.txt#L87
But yes maybe it's a mistake in CMakeLists

Reply:
At least there is no dependence on any external tool in the project's header:
https://github.com/boost-ext/sml/blob/b6b1008a31b67479bcd99bf31e36afea1dd4df6f/include/boost/sml.hpp

Reply:
Yes you're right, no boost dependency.
Upstream CMakeLists is misleading.